pkg/lib/cockpit-components-terminal.jsx

```diff
@@ -94,8 +100,10 @@
         },
 
         render: function () {
+            var cls = 'console-ct';
+
             // ensure react never reuses this div by keying it with the terminal widget
-            return <div ref="terminal" className="console-ct" key={this.state.terminal} />;
+            return <div ref="terminal" className={cls} key={this.state.terminal} />;
```

Comment:
What's the purpose of the new `cls`? As far as I can see it's being used just once, right here. Debugging leftover?